afs/afs-ext-base/src/main/java/com/powsybl/afs/ext/base/AbstractModificationScript.java

```diff
@@ -1,88 +1,11 @@
-/**
- * Copyright (c) 2018, RTE (http://www.rte-france.com)
- * This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/.
- */
 package com.powsybl.afs.ext.base;
 
-import com.google.common.io.CharStreams;
-import com.powsybl.afs.ProjectFile;
 import com.powsybl.afs.ProjectFileCreationContext;
-import com.powsybl.afs.storage.events.AppStorageListener;
-import com.powsybl.afs.storage.events.NodeDataUpdated;
-import com.powsybl.afs.storage.events.NodeEvent;
-import com.powsybl.afs.storage.events.NodeEventType;
 
-import java.io.*;
-import java.nio.charset.StandardCharsets;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Objects;
-
-/**
- * @author Geoffroy Jamgotchian <geoffroy.jamgotchian at rte-france.com>
- */
-public abstract class AbstractModificationScript extends ProjectFile implements StorableScript {
-
-    private final String scriptContentName;
-
-    private final List<ScriptListener> listeners = new ArrayList<>();
-
-    private final AppStorageListener l = eventList -> processEvents(eventList.getEvents(), info.getId(), listeners);
+public abstract class AbstractModificationScript extends AbstractScript {
```

Comment:
I didn't understand the use of this class, it has only one subclass, why ModificationScript didn't inherit from AbstractScript directly? even you want to keep this class to ovoid breaking changes you can put it @deprecated and make ModificationScript inherit from AbstractScript.